docs/_components/search.md

```diff
@@ -10,12 +10,12 @@ lead: >
 ---
 
 ## Default
-Search should be easily found. For the default search version, consider displaying in the header and/or side navigation. 
+Search should be easily found. For the default search version, consider displaying in the header and/or side navigation.
 
 {% capture example %}
 <form accept-charset="UTF-8" action="https://search.usa.gov/search" class="usa-search usa-search--smalldisplay-flex flex-justify-center tablet:grid-col-4" method="get" role="search">
   <input name="utf8" type="hidden" value="&#x2713;"/>
-  <input name="affiliate" type="hidden" value="login.gov"/>
+  <input name="affiliate" type="hidden" value="Login.gov"/>
```

Comment:
Separately, we might want to update this example so it's less hard-coded to the specific usage of search.gov on the brochure site. In the meantime, the value is possibly meaningful for how search.gov tracks our usage, or at least isn't shown visually to the user, so I might caution against changing it.

```suggestion
  <input name="affiliate" type="hidden" value="login.gov"/>
```